llarp/dht/messages/gotname.cpp

```diff
@@ -0,0 +1,49 @@
+#include <dht/messages/gotname.hpp>
+#include <lokimq/bt_serialize.h>
+
+namespace llarp::dht
+{
+  constexpr size_t NameSizeLimit = 512;
```

Comment:
as per conversation about this, seems checking against this limit doesn't make sense here.